Mono.Debugging.Soft/PortablePdbData.cs

```diff
@@ -130,8 +130,8 @@ public SourceLink GetSourceLink (string originalFileName)
 
 		SourceLinkMap[] GetSourceLinkMaps ()
 		{
-			using (var fs = stream)
-			using (var provider = MetadataReaderProvider.FromPortablePdbStream (fs)) {
+			using (stream)
+			using (var provider = MetadataReaderProvider.FromPortablePdbStream (stream)) {
```

Comment:
By default, `FromPortablePdbStream` disposes of the underlying stream. `using (stream)` seems unneeded.

Do we need to pre-fetch metadata to avoid later IO?

Reply:
> Do we need to pre-fetch metadata to avoid later IO?

This might fetch more data than we need. I think this option is for users who want to have full control over the stream after reading.